### PR TITLE
[FLOC-1999] Vagrant gsg floc 1999

### PR DESCRIPTION
**Pull request by adamtheturtle**
Fixes https://clusterhq.atlassian.net/browse/FLOC-1999

### CODE REVIEW

File: docs/gettingstarted/index.rst [hunk 1]
```diff
@@ -162,11 +162,9 @@ Getting started with Flocker
 
          Simulate a Flocker cluster with virtual machines on your laptop (requires `Vagrant <http://www.vagrantup.com/downloads>`_, `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_):
 
-         .. code-block:: console
+         .. version-code-block:: console
 
-            you@laptop:~$ git clone \
-              https://github.com/clusterhq/vagrant-flocker && \
-              cd vagrant-flocker && \
+            you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile && \
```
Comment (exarkun) [inline docs/gettingstarted/index.rst]: The old approach gave you a directory dedicated to the vagrant stuff being created.  That might have been a nice thing.  If you agree, I suggest inserting `mkdir vagrant-flocker && cd vagrant-flocker && ...` at the front of this and the following command.

**Comment by exarkun** [comment]
Thanks.  This looks great.  Please address one inline comment (if you think it makes sense) and then merge.

**Comment by exarkun** [comment]
When you merge this, please also add a note to the flocker-vagrant README saying the repository is obsolete.

File: docs/gettingstarted/index.rst (hunk 1 above)
Comment (adamtheturtle) [inline docs/gettingstarted/index.rst]: I don't agree, so I won't do this.

**Comment by adamtheturtle** [comment]
I added a note to the README at https://github.com/ClusterHQ/vagrant-flocker/blob/master/README.rst.

**Pull request merged**